torch/python/tests/test_e3nn.py

```diff
@@ -0,0 +1,71 @@
+import itertools
+
+import pytest
+
+import sphericart_torch
+import torch
+
+
+# only run e3nn tests if e3nn is present
+try:
+    import e3nn
+
+    _HAS_E3NN = True
+except ModuleNotFoundError:
+    _HAS_E3NN = False
+
+torch.manual_seed(0)
+_PRECISION = 1e-8
```

Comment:
```suggestion
TOLERANCE = 1e-8
```

this is quite a high tolerance for float64. Do we really not do better than this?